aws_account.go

```diff
@@ -22,6 +25,11 @@ type AWSAccount struct {
 
 // ParseAWSAccounts extract the aws accounts from the saml assertion
 func ParseAWSAccounts(samlAssertion string) ([]*AWSAccount, error) {
+        decSamlAssertion, _ := b64.StdEncoding.DecodeString(samlAssertion)
+        if strings.Contains(string(decSamlAssertion), "signin.amazonaws.cn") {
```

Comment:
Can you just add a log message to inform the operator this login server has been selected, this will also help me debug issues

Reply:
@wolfeidau  Thank you for your comment, I've added below `30 line`..

One thins to share, my change is working properly when I execute it using below command.
```
AWS_REGION=cn-north-1 saml2aws login
```